source/MaterialXCore/Definition.h

```diff
@@ -242,6 +242,29 @@ class MX_CORE_API Implementation : public InterfaceElement
         return getAttribute(FUNCTION_ATTRIBUTE);
     }
 
+    /// @}
+    /// @name Nodegraph String
+    /// @{
+
+    /// Set the nodegraph string for the Implementation.
+    void setNodeGraph(const string& function)
```

Comment:
Nitpick: the parameter could be named "nodegraph" instead for "function"

Reply:
Thanks, probably a cut-paste error.